src/app/src/main/kotlin/org/imperial/mrc/hint/APIClient.kt

```diff
@@ -21,7 +21,7 @@ open class HintAPIClient(
 
         val json = objectMapper.writeValueAsString(
                 mapOf("type" to type.toString().toLowerCase(),
-                        "path" to path))
+                        "path" to "/uploads/$path"))
```

Comment:
Why not include /uploads in the configured path?

Reply:
At the moment hintr has no configured upload path, hence passing an absolute path here. See discussion here: https://github.com/mrc-ide/hintr/pull/7